sdk/storage/azure-storage-blobs/swagger/README.md

```diff
@@ -102,9 +102,13 @@ directive:
           "name": "ApiVersion",
           "modelAsString": false
           },
-        "enum": ["2021-12-02"],
+        "enum": ["2022-11-02"],
         "description": "The version used for the operations to Azure storage services."
       };
```

Comment:
The transform at line 108 confuses me. Why is it necessary given that the enum value is set at line 105?

Is this to fix an authoring error in the source swagger document? If so, why isn't it being fixed at the source so it works for all languages? Why does this have to be fixed for C++ only?

Reply:
Somewhat related to the previous comment. At line 105, the enum value is to generate ApiVersion constant and at line 108, the value is to specify the `ApiVersion` in rest_client.cpp

Reply:
And sometimes the version of swagger is not consistent with the actual ApiVersion, for example, the latest version of the blob in [swagger](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-12-02/blob.json) is 2021-12-02, but the current version required for the feature is 2022-11-02. So we have to specify the version manually.